.env.example

```diff
@@ -103,3 +103,6 @@ FORCE_COLOR=1
 # Do not group published resources by outline item id.
 # repository/id/resourceId* instead of repository/id/outlineId/resourceId*
 FLAT_REPO_STRUCTURE=1
+
+# tce-api-video API key
+TCE_API_VIDEO_API_KEY=
```

Comment:
As discussed, please remove tce specific secrets from `.env.example` and keep them in the tce's readme.